stdlib/2and3/ensurepip/__init__.pyi

```diff
@@ -0,0 +1,3 @@
+
+def version() -> str: ...
+def bootstrap(*, root: str = ..., upgrade: bool = ..., user: bool = ..., altinstall: bool = ..., default_pip: bool = ..., verbosity: int = ...) -> None: ...
```

Comment:
Unfortunately we need to use a conditional for Python 2, since the arguments can be provided positionally there.

Also, root can be `None`.